pylint/lint/message_state_handler.py

```diff
@@ -411,9 +412,14 @@ def process_tokens(self, tokens: list[tokenize.TokenInfo]) -> None:
                         try:
                             meth(msgid, "module", l_start)
                         except exceptions.UnknownMessageError:
```

Comment:
I just wonder, there are more place where we catch this exception. Should we also add this special handling there?

Reply:
Well, something really elegant would be to reraise a ``DeletedMessageError`` with the justification...